atlite/data.py

```diff
@@ -42,14 +43,18 @@ def get_features(cutout, module, features, tmpdir=None):
             cutout, feature, tmpdir=tmpdir, lock=lock, **parameters
         )
         datasets.append(feature_data)
-
     datasets = compute(*datasets)
-
     ds = xr.merge(datasets, compat="equals")
+    # for v in ds:
+    #     ds[v].attrs["module"] = module
+    #     fd = datamodules[module].features.items()
+    #     ds[v].attrs["feature"] = [k for k, l in fd if v in l].pop()
+    fd = datamodules[module].features.items()
+    datavars = list(chain(*[l for k, l in fd]))
     for v in ds:
         ds[v].attrs["module"] = module
-        fd = datamodules[module].features.items()
-        ds[v].attrs["feature"] = [k for k, l in fd if v in l].pop()
+        if v in datavars:
+            ds[v].attrs["feature"] = [k for k, l in fd if v in l].pop()
```

Comment:
The CMIP data have longitude bins, latitude bins and time bins included as variables and these aren't defined as variables in the features dict. If I don't include if statement it will try to pop an empty list. Suggestion on a better way to do this?
```
Data variables:
    time_bnds    (time, bnds) datetime64[ns] 2021-01-01 ... 2021-02-01
    lat_bnds     (y, bnds) float64 31.65 32.35 32.65 33.35 ... 82.35 82.65 83.35
    lon_bnds     (x, bnds) float64 346.6 347.4 347.6 348.4 ... 44.35 44.65 45.35
    wnd10m       (time, y, x) float32 dask.array<chunksize=(100, 52, 59), meta=np.ndarray>
    influx       (time, y, x) float32 dask.array<chunksize=(100, 52, 59), meta=np.ndarray>
    outflux      (time, y, x) float32 dask.array<chunksize=(100, 52, 59), meta=np.ndarray>
    temperature  (time, y, x) float32 dask.array<chunksize=(100, 52, 59), meta=np.ndarray>
    runoff       (time, y, x) float32 dask.array<chunksize=(100, 52, 59), meta=np.ndarray>

```

Reply:
I'd rather question whether we want to carry these variables along? If not needed, I'd argue to delete them before (in get_data) since they not requested from the feature list anyway